src/content/docs/en/reference/experimental-flags/fonts.mdx

```diff
@@ -0,0 +1,718 @@
+---
+title: Experimental fonts API
+sidebar:
+  label: Fonts
+i18nReady: true
+tableOfContents:
+  minHeadingLevel: 2
+  maxHeadingLevel: 6
+---
+
+import Since from '~/components/Since.astro';
+import ReadMore from '~/components/ReadMore.astro';
+import { Steps, Tabs, TabItem } from '@astrojs/starlight/components';
+
+<p>
+
+**Type:** `FontFamily[]`<br />
+<Since v="5.7.0" />
+</p>
+
+This experimental feature allows you to use fonts from your filesystem and various font providers (eg. Google, Fontsource, Bunny) through a unified, fully customizable and type-safe API.
+
+Web fonts can impact page performance at both load time and rendering time. This API helps you keep your site performant with automatic [web font optimizations](https://web.dev/learn/performance/optimize-web-fonts) including preload links, optimized fallbacks, and opiniated defaults to avoid common problems such as downloading unnecessary font files.
```

Comment:
```suggestion
Web fonts can impact page performance at both load time and rendering time. This API helps you keep your site performant with automatic [web font optimizations](https://web.dev/learn/performance/optimize-web-fonts) including preload links, optimized fallbacks, and opinionated defaults to avoid common problems such as downloading unnecessary font files.
```
a last minute typo fix from before that didn't get pushed!